woocommerce/woocommerce-android
# Integrate Automattic about screen

Closes: #5435


Merge Instructions:
1. Make sure https://github.com/Automattic/about-automattic-android/pull/29 is merged
2. Update hashcode
3. Remove "Do not merge" label
4. Review this PR
5. Merge this PR

### Description


This PR replaces the existing About screen with the unified About screen which is used across Automattic apps.

### Testing instructions

1. Tap on My Store
2. Tap on the gear icon in the top right corner
3. Scroll down and tap on "WooCommerce" row under the "About the app" section
4. Notice the new about screen is shown
5. Test clicking on all the rows and make sure they work as expected
6. Verify that "view shown" and "button click" events are being tracked

### Images/gif

| Before | After |
| --- | --- |
| ![Screenshot_1640776766](https://user-images.githubusercontent.com/2261188/147657088-3ec2cd5a-2f07-46d4-804f-c0084011e4eb.png) | ![30-top-dark](https://user-images.githubusercontent.com/2261188/147657000-7ad46a3f-2285-4153-ab1c-f89c35757713.png) |
| | ![30-bot-dark](https://user-images.githubusercontent.com/2261188/147657004-1defcf58-be61-484b-939d-eb6f6832f349.png) |
| ![Screenshot_1640776772](https://user-images.githubusercontent.com/2261188/147657090-0fc3ee2f-3378-4d21-9cdc-7539817070e5.png) | ![30-top-light](https://user-images.githubusercontent.com/2261188/147657003-f3d92569-4a98-4e12-9974-5babe88b710e.png) |
| | ![30-bot-light](https://user-images.githubusercontent.com/2261188/147656998-0a31ce7b-da8e-4364-9780-d8cb2eb0bde1.png) |

P.S. I considered adding unit tests but decided it's not worth the effort. About screen is static and we probably won't touch it for years.














cc @ravishanker
